content/working_with/official_plugins/Orchestration/helm.md

```diff
@@ -0,0 +1,465 @@
+* Content of Kubernetes config file (YAML).
+
+
+**Example 1:**
+
+This is an example for authentication with kubeconfig file content:
+
+{{< highlight  yaml  >}}
+ inputs:
+
+  configuration_file_content:
+    type: string
+
+node_templates:
+  
+  release:
+    type: cloudify.nodes.helm.Release
+    properties:
+      client_config:
+        configuration:
+          file_content: {get_input: configuration_file_content } # secret also can be used.
+      resource_config:
+        name: "myrelease"
+        chart: { concat: [ { get_input: repo_name },'/', { get_input: chart_name } ] }
+    relationships:
+      - target: helm_install
+        type: cloudify.helm.relationships.run_on_host
+      - target: repo
+        type: cloudify.relationships.depends_on
+
+{{< /highlight >}}
+
+
+**Example 2:**
+
+This is another example for authentication with kubeconfig file content,as a dict:
+
+{{< highlight  yaml  >}}
+ 
+ node_templates:
+  
+  release:
+    type: cloudify.nodes.helm.Release
+    properties:
+      client_config:
+        configuration:
+          file_content:
+            apiVersion: v1
+            kind: Config
+            preferences: {}
+            current-context: kubernetes-admin@kubernetes
+            clusters:
+            - name: kubernetes
+              cluster:
+                certificate-authority-data: { get_input: kubernetes_certificate_authority_data }
+                server: { concat: [ 'https://', { get_input: kubernetes_master_ip}, ':', { get_input: kubernetes_master_port } ] }
+            contexts:
+            - name: kubernetes-admin@kubernetes
+              context:
+                cluster: kubernetes
+                user: kubernetes-admin
+            users:
+            - name: kubernetes-admin
+              user:
+                client-certificate-data: { get_input: kubernetes-admin_client_certificate_data }
+                client-key-data:{ get_input: kubernetes-admin_client_key_data }
+      resource_config:
+        name: "myrelease"
+        chart: { concat: [ { get_input: repo_name },'/', { get_input: chart_name } ] }
+    relationships:
+      - target: helm_install
+        type: cloudify.helm.relationships.run_on_host
+      - target: repo
+        type: cloudify.relationships.depends_on
+
+{{< /highlight >}}
+
+**Example 3:**
+
+This is an example for authentication with kubeconfig file path(the file is in the blueprint archive):
+
+{{< highlight  yaml  >}}
+
+node_templates:
+  
+  release:
+    type: cloudify.nodes.helm.Release
+    properties:
+      client_config:
+        configuration:
+          blueprint_file_name: path/to/kubeconfig
+      resource_config:
+        name: "myrelease"
+        chart: { concat: [ { get_input: repo_name },'/', { get_input: chart_name } ] }
+    relationships:
+      - target: helm_install
+        type: cloudify.helm.relationships.run_on_host
+      - target: repo
+        type: cloudify.relationships.depends_on
```

Comment:
The examples look good. You don't need to show relationships or resource config for the authentication examples.